client-misk-jooq/README.md

```diff
@@ -0,0 +1,3 @@
+# client-misk-jooq
+
+Jooq backfila client backend implementation for misk apps. 
```

Comment:
Is there actually any reason to make this misk specific?

Reply:
I think the other clients are. If we do this outside of misk, we need to remove all the misk deps and see if everything works. Maybe something for another day.

Reply:
I want to do a larger project where you can have different client service implementations like misk, servicecontainer and others. I think it is probably fine for now?

Reply:
yea this is fine for now. I would like to separate it from misk at some point, there's very little anyway